Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG ATLANTIS_BASE_TAG_TYPE=alpine
 
 # Stage 1: build artifact
 
-FROM golang:1.19.5-alpine AS builder
+FROM golang:1.20-alpine AS builder
```

Comment:
Wouldn't this use semantic versioning?

```suggestion
FROM golang:1.20.0-alpine AS builder
```

Reply:
yeah, 1.20 would be pointing to the latest 1.20.x, but I can take it.